Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -1,20 +1,88 @@
 /**
-*  Publish
-*  Copyright (c) John Sundell 2019
-*  MIT license, see LICENSE file for details
-*/
+ *  Publish
+ *  Copyright (c) John Sundell 2019
+ *  MIT license, see LICENSE file for details
+ */
 
 import Foundation
 import Files
 import ShellOut
 
+#if canImport(FileWatcher)
+import FileWatcher
+#endif
+
 internal struct WebsiteRunner {
+    static let nanosecondsPerSecond: UInt64 = 1_000_000_000
```

Comment:
Workaround for `NSEC_PER_SEC` not being available on Linux which according to [this](https://forums.swift.org/t/nsec-per-sec-is-an-unresolved-identifier-on-linux/34991) is a bug. Alternatively, I could have wrapped the NSEC_PER_SEC usage in a `#if canImport(FileWatcher)` condition, since it's only used alongside FileWatcher.